.clinerules/05-workflow.md

```
@@ -0,0 +1,55 @@
+
+## コミットのタイミング
+
+以下のタイミングでコミットを提案すること:
+
+- 1つの機能・修正が完成したとき
+- テストがパスしたとき
+- リファクタリングが完了したとき
+
+コミットメッセージは以下の形式を使う:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)